### DISCUSSION TITLE
I can't run start WalletConnectKotlinV2, Android Studio tip: File google-services.json is missing

### DISCUSSION THREAD

**Discussion by DaZhaoGuo** [WalletConnect/walletconnect-monorepo#3340] [Developer Support] [answered]
Error:
Execution failed for task ':sample:dapp:processDebugGoogleServices'.
> File google-services.json is missing. The Google Services Plugin cannot function without it. 
   Searched Location: 
  /Users/guo/workspace/wallet/walletConnect/WalletConnectKotlinV2/sample/dapp/src/debug/google-services.json
  /Users/guo/workspace/wallet/walletConnect/WalletConnectKotlinV2/sample/dapp/src/google-services.json
  /Users/guo/workspace/wallet/walletConnect/WalletConnectKotlinV2/sample/dapp/src/Debug/google-services.json
  /Users/guo/workspace/wallet/walletConnect/WalletConnectKotlinV2/sample/dapp/google-services.json

--------
How can I run sample/dapp

**Comment by DaZhaoGuo** [comment]
Our android app want use WalletConnect to connect MetaMask

**Comment by Talhaali00** [comment, marked as answer]
Hello @bagguo,
To run the sample applications, you will need to setup a Firebase project and drop the google-services.json file into the sample project as per their [documentation](https://firebase.google.com/docs/android/setup#console)

**Comment by DaZhaoGuo** [comment]
您好，您的邮件已收到，谢谢！
